ui/listener-api/src/main/scala/pl/touk/nussknacker/ui/listener/DeploymentComment.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.ui.listener
+
+import cats.data.Validated
+import cats.data.Validated.{Invalid, Valid}
+
+import scala.language.implicitConversions
+
+trait Comment {
+  def value: String
+}
+
+case class DeploymentComment private(value: String) extends Comment {
+
```

Comment:
I would like to have as small amount of code as possible in api module. Let's move all other stuff to place of usage (UI). e.g. those methods are used only in `DbProcessActionRepository`. IMO `withPrefix` only has potential to be used in other places

Reply:
i understand, moved them to places of usage, should be better now

Reply:
I'm thinking about moving other parts: content of `DeploymentComment` and `DeploySettings` into UI as well. I understand your concept, that thanks to current approach it is quite hard to create comment which not matches pattern from settings, but on the other hand we have some implementation logic of those validations in API, so someone can rely on it after changes in this logic we should mention this as a breaking change of our API. Also will be hard to do some refactors in settings. E.g. now we have `deploySettings` which are only related to comment checking. What if we want to other deploy settings that are unrelated to comments? We will probable rename this class, add some nesting etc. which will cause another breakage of API compatibility.